README.md

```diff
@@ -22,11 +22,15 @@ Application Insights logging adapters.
 
 If you use NLog, log4Net or System.Diagnostics.Trace for diagnostic tracing in your  application, you can have your logs sent to Application Insights, where you can explore and search them. Your logs will be merged with the other telemetry coming from your application, so that you can identify the traces associated with servicing each user request, and correlate them with other events and exception reports.
 
-[Application Insights Documentation](https://azure.microsoft.com/en-us/documentation/articles/app-insights-search-diagnostic-logs/#trace).
+Read more:
+- [Microsoft Docs: "Explore .NET trace logs in Application Insights"](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-asp-net-trace-logs)
```

Comment:
remove `en-US` from urls so there will be language redirect for people outside of US